### DISCUSSION TITLE
Avoid users without FQDN can write me

### DISCUSSION THREAD

**Discussion by openaspace** [modoboa/modoboa#2658] [General]
Hello after 2 days the spam is starting...
How I can avoid to receive email from domain without FQDN?

Thanks.

**Comment by Spitfireap** [comment]
Hi, 
If you used the installer, you should have this in ``/etc/postfix/main.cf`` : 

```
smtpd_recipient_restrictions =
      check_policy_service inet:127.0.0.1:9999
      permit_mynetworks
      permit_sasl_authenticated
      check_recipient_access
          proxy:pgsql:/etc/postfix/sql-maintain.cf
          proxy:pgsql:/etc/postfix/sql-relay-recipient-verification.cf
      reject_unverified_recipient
      reject_unauth_destination
      reject_non_fqdn_sender
      reject_non_fqdn_recipient
      reject_non_fqdn_helo_hostname
```
The last 3 parameters are basically doing what you're asking (example https://www.postfix.org/postconf.5.html#reject_non_fqdn_helo_hostname). Or are you looking for something else ?
